ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/page/wizard/project/NewLiferayModulesExtWizard.java

```diff
@@ -27,6 +27,10 @@ public NewLiferayModulesExtWizard(SWTBot bot) {
 		super(bot, 2);
 	}
 
+	public ToolbarButtonWithTooltip getAddOriginMoudleBtn() {
```

Comment:
let's rename to getOverrideOriginalModuleFilesBtn or getAddFildesFromOriginalModuleBtn